client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -175,16 +119,22 @@ const OutboundShipmentListViewComponent: FC = () => {
     []
   );
 
-  const { table, selectedRows, resetRowSelection } =
+  const { table, selectedRows } =
     usePaginatedMaterialTable<OutboundRowFragment>({
-      tableId: 'outbound-shipment-list-view',
+      tableId: 'outbound-shipment-list',
       isLoading,
       onRowClick: row => navigate(row.id),
       columns: mrtColumns,
       data: data?.nodes ?? [],
       totalCount: data?.totalCount ?? 0,
       initialSort: { key: 'invoiceNumber', dir: 'desc' },
       getIsRestrictedRow: isOutboundDisabled,
+      renderEmptyRowsFallback: () => (
+        <NothingHere
+          body={t('error.no-outbound-shipments')}
+          onCreate={modalController.toggleOn}
```

Comment:
I put in a default `NothingHere`, but overriding here, as per the old table below, to get the custom body text and `create a new one` handler for outbound shipments